src/websocket.rs

```diff
@@ -85,6 +183,9 @@ impl AdminWebsocket {
                 if let Ok(key) = AgentPubKey::from_raw_39(key_vec) {
                     info!("returning agent key from file");
                     self.agent_key = Some(key.clone());
+                    // if using devNet,
+                    // enable membrane proof using generated key
+                    self.enable_memproof_dev_net();
```

Comment:
you don't need to generate it here, this is where is just reading from a local copy. in the line below you have already created the mem-proof when the key was generated